Dockerfile

```diff
@@ -1,12 +1,20 @@
-FROM alpine:edge
+FROM node:8.2-alpine
 
-WORKDIR /code
-ADD . /app
+WORKDIR /usr/src/app
+
+COPY package.json ./
 
-RUN apk --update add nodejs curl && \
-    npm install -g npm && \
-    cd /app && \
+RUN apk --update add curl && \
```

Comment:
Do we need curl after installing the packages?

If so, can you install it in its own layer before the `package.json` copy for better caching?

If not, can you uninstall it in this layer?